nixos/modules/programs/fzf.nix

```diff
@@ -1,18 +1,22 @@
 { pkgs, config, lib, ... }:
 
-with lib;
-
 let
+  inherit (lib) maintainers;
+  inherit (lib.meta) getExe;
+  inherit (lib.modules) mkIf mkRemovedOptionModule;
+  inherit (lib.options) mkEnableOption;
+  inherit (lib.strings) optionalString;
```

Comment:
Please take the functions from lib. Where they are exactly is an implementation detail and just complicates things here.

Also we would avoid maintaining this list of functions by just prefixing the functions with lib.

Reply:
>  Where they are exactly is an implementation detail

Being explicit is always better. And as we know there are many functions not even exposed at the top level or have the same name in multiple scopes.